docs/tutorials/ASR.md

```
@@ -0,0 +1,67 @@
+| **Dynamic Operation** | Use of the ADS-DV within an approved Dynamic Operating Area under the defined ASR and area-control rules. |
+
+
+## Brief:
+
+According to the IMechE Documentation, the role of ASR is defined as follows:
+
+**"The responsible operator who supervises the ADS-DV and operates the RES."**
+
+I'm going to designate numbers for these rules, not because they are provided, but because they might make them easier to remember.
+
+#### Rule 1:
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)